src/KafkaClient/Connections/ISslConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net.Security;
+
+namespace KafkaClient.Connections
+{
+    public interface ISslConfiguration
```

Comment:
Is there a simple, default implementation that easily enables ssl? That'd be super helpful for getting started.